trusted_applet/ssh.go

```diff
@@ -132,6 +132,14 @@ func startSSHServer(ctx context.Context, listener net.Listener, addr string, por
 
 	srv.AddHostKey(signer)
 
+	chansToClose := []*ssh.ServerConn{}
```

Comment:
Nit, the name looks a bit odd because NewServerConn returns a connection, two channels and an error, but chansToClose doesn't contain any of these channels. Maybe rename it to connsToClose?